examples/with-thirdpartyemailpassword-2fa-passwordless/src/App.tsx

```diff
@@ -78,6 +78,37 @@ SuperTokens.init({
                             </div>
                         );
                     },
+                    // we override the component which shows the change phone number button
+                    PasswordlessUserInputCodeFormFooter_Override: ({ DefaultComponent, ...props }) => {
+                        // the logic here is very similar to the previous step's logic
+                        let [showDefaultUI, setShowDefaultUI] = useState(false);
+                        useEffect(() => {
+                            Session.getAccessTokenPayloadSecurely()
+                                .then(async (accessTokenPayload) => {
+                                    let phoneNumber = accessTokenPayload.phoneNumber;
+                                    if (phoneNumber === undefined) {
+                                        // since the phone number does not exist in the access token's
+                                        // payload, we want to show the button
+                                        setShowDefaultUI(true);
+                                    }
+                                })
+                                .catch((err) => {
+                                    // it can come here if a session doesn't exist.
+                                    // in this case, the screen we should redirect to the
+                                    // first login challenge
+                                    SuperTokens.redirectToAuth({
+                                        redirectBack: false,
+                                    });
+                                });
```

Comment:
You can change this to call doesSessionExist and if it doesn't call 
```
SuperTokens.redirectToAuth({
                                        redirectBack: false,
                                    });
```

else do the accessTokenPayload thing.

The reason it was done like this previously was cause we had overriden doesSessionExist to return false even if it did exist and so the easiest way to know if it actually existed was by calling accessTokenPayload and it throwing an error.